src/app/(pages)/sponsors/page.tsx

```diff
@@ -1,7 +1,85 @@
-export const metadata = {
-  title: "SESA - Sponsors",
+import React from "react";
+import OurSponsors from "@/components/OurSponsors";
+import StatisticsSection from "@/components/SponsorStats";
+import SponsorUsCards from "@/components/SponsorUsCards"; // Import the new component
+
+const goldSponsors = [
+  {
+    name: "Jane Street",
+    logo: "sponsors-images/jane-st.png",
+    link: "https://www.janestreet.com/join-jane-street/overview/",
+  },
+];
+
+const silverSponsors = [
+  {
+    name: "IMC Trading",
+    logo: "sponsors-images/IMC.png",
+    link: "https://careers.imc.com/us/en",
+  },
+  {
+    name: "Atlassian",
+    logo: "sponsors-images/atlassian.png",
+    link: "https://www.atlassian.com/company/careers",
+  },
+  {
+    name: "Canva",
+    logo: "sponsors-images/Canva.svg",
+    link: "https://www.lifeatcanva.com/en/jobs/",
+  },
+  {
+    name: "Serato",
+    logo: "sponsors-images/serato.png",
+    link: "https://serato.com/careers",
+  },
+  {
+    name: "Vista",
+    logo: "sponsors-images/vista.png",
+    link: "https://www.vista.co/careers/",
+  },
+  {
+    name: "Sandfield",
+    logo: "sponsors-images/sandfield.png",
+    link: "https://www.sandfield.co.nz/careers/interns-graduates",
+  },
+  {
+    name: "Emergency Q",
+    logo: "sponsors-images/EmergencyQ.svg",
+    link: "https://www.emergencyq.com/",
+  },
+];
+
+const SponsorsPage: React.FC = () => {
+  return (
+    <div className="page-padding bg-gray-900">
```

Comment:
Change to light default background colour and change all other text/colours accordingly